src/config/options.schema.json

```diff
@@ -963,6 +963,30 @@
                 "Do widening on contexts. Keeps a map of function to call state; enter will then return the widened local state for recursive calls.",
               "type": "boolean",
               "default": false
+            },
+            "ctx_gas": {
+              "title": "ana.context.ctx_gas",
+              "description": "If enabled, the first x recursions of each path are analyzed context-sensitive, the remaining context-insensitive. x denotes the context gas, which is defined in ana.context.ctx_gas_value.",
+              "type": "boolean",
+              "default": false
+            },
+            "ctx_gas_value": {
+              "title": "ana.context.ctx_gas_value",
+              "description": "Number of calls that are analyzed context-sensitive. Used for the ContextGasLifter, which is enabled with ana.context.ctx_gas. Must be >= 0!",
+              "type": "integer",
+              "default": 10
+            },
```

Comment:
These should be one option like `ana.context.gas` with the numeric value (by default 0), where 0 would bean it's deactivated.
We have other options that work similarly.

Reply:
This would take away the option to configure a gas value of `0`, which is equivalent to an context-insensitve analysis. If one don't want this "fast configureable"  context-insensitive option for context gas, I can configure it as you mentioned.

Reply:
Sounds reasonable. The special disabled value could just be -1 then. Actually, that's what `incremental.restart.sided.fuel` even does, which is also fuel/gas for another purpose.

Reply:
That sounds good!  
I update it in the code